data-access/src/app/domain/contexts/cases/service-ticket/v1/service-ticket-v1.ts

```diff
@@ -239,6 +253,16 @@ export class ServiceTicketV1<props extends ServiceTicketV1Props> extends Aggrega
     this.props.setRequestorRef(requestor);
   }
 
+  set AssignedVendor(assignedVendor: VendorUserEntityReference) {
+    if (!this.isNew) {
+      throw new Error('Unauthorized');
+    }
+    if (!assignedVendor) {
```

Comment:
Remove this, assignedVendor can be null since it can be unset. We can remove a vendor user from a ticket.